cpp/ql/lib/semmle/code/cpp/models/implementations/StructuredExceptionHandling.qll

```diff
@@ -1,9 +1,11 @@
 import semmle.code.cpp.models.interfaces.Throwing
 
-class WindowsDriverFunction extends ThrowingFunction {
-  WindowsDriverFunction() {
+class WindowsDriverExceptionAnnotation extends ThrowingFunction {
+  WindowsDriverExceptionAnnotation() {
```

Comment:
This is also technically a breaking change, but I really doubt that anyone depends on this name specifically so that's probably fine. However, is there a reason not to keep the old name?